cmd/pd-server/main.go

```diff
@@ -68,7 +67,7 @@ func main() {
 		log.Fatal("initialize logger error", zap.Error(err))
 	}
 	// Flushing any buffered log entries
-	defer log.Sync()
+	defer log.Sync() //nolint:errcheck
```

Comment:
It seems we don't need this comment since we have already disabled `errcheck`?

Reply:
It's just because my local golangci-lint configuration is stricter.